tests/plugins/wysiwygarea/manual/detached.html

```diff
@@ -0,0 +1,48 @@
+<button id="toggleButton1">Toggle</button>
+<button id="setButton">Set data</button>
+
+<div>
+	<div id="editor">
+		<p>Lorem ipsum dolor sit amet.</p>
+	</div>
+</div>
+
+
+<div id="fakedId"></div>
+<div id="fakedId1"></div>
+
+
+<script>
+	var editorElement = document.getElementById("editor");
+
+	var textArea = document.createElement("textarea");
+
+	editorElement.appendChild(textArea);
+
+
+	var e = CKEDITOR.replace( textArea, {
+		on: {
+			instanceReady:function(evt){}
+		}
+	});
+	e.setData("<p>ole!</p>");
+
+	var btn = CKEDITOR.document.getById("toggleButton1");
+	var setBtn = CKEDITOR.document.getById("setButton");
+
+	setBtn.on ('click', function() {
+		e.setData('any any any');
+	});
+
+	var t = false;
```

Comment:
Is that possible to give this variable a meaningful name for better intuition? Maybe `editorDetached`?